tests/suites/test_suite_ssl.function

```diff
@@ -3759,6 +3846,36 @@ void handshake_version( int version, int dtls )
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_X509_CRT_PARSE_C:!MBEDTLS_USE_PSA_CRYPTO:MBEDTLS_PKCS1_V15:MBEDTLS_RSA_C:MBEDTLS_ECP_DP_SECP384R1_ENABLED:MBEDTLS_SSL_PROTO_TLS1_1:MBEDTLS_SSL_PROTO_TLS1_2 */
+void handshake_minmax_version( int client_min_version, int client_max_version, int server_min_version, int server_max_version, int expected_version )
```

Comment:
Instead of adding a new function we could simply rewrite `handshake_version()`. All of the tests using the old `handshake_version` seem expressible by `handshake_minmax_version()`.